### PR TITLE
Add `.env` file to set `ALLOWED_RAM`

### PR DESCRIPTION
**Pull request by harshsharma2002**
- Add a .env file, to dynamically change the value of the `ALLOWED_RAM` depending on the language.

### CODE REVIEW

**Comment by anipnwr7777** [comment]
Hey @harshsharma2002, please don't remove `.env` from `.gitignore`, that could potentially expose secret information publicly.
Don't depend on version control for `.env` to be copied to docker for building the image.
Explore other options on how to set env.

**Comment by harshsharma2002** [comment]
Hey @anipnwr7777 , Just confirming my idea, we can copy the `.env` file as `.env.example` file to the docker container having the .env in the ignore file and then change the name of the `.env.example` to `.env` this will load `.env` and also have it in docker container's ignore file, this should solve the problem right.

an alternative solution will be to copy the `.env` file after the docker container is up and running manually which will allow us to bypass the ignore statement.

**Comment by anipnwr7777** [comment]
No this doesn't solve the problem. In this solution, someone may end up exposing their keys from  `.env.example` itself.
For now don't worry about `.env` file, just pass `ALLOWED_RAM` as part of docker run command as we are passing `OPENAI_API_KEY` and also update the README with that.

File: .dockerignore [hunk 1]
```diff
@@ -1 +0,0 @@
-.env
```
Comment (anipnwr7777) [inline .dockerignore:1]: we don't need this, revert

File: envloader.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const fs = require('fs')
 
 const initializeENV = () => {
-    const path = '/env/.env'
+    const path = '/usr/bin/.env'
```
Comment (anipnwr7777) [inline envloader.js:4]: don't need this, please revert.

**Review by anipnwr7777 — CHANGES_REQUESTED** [review]
please fix the changes.

**Pull request closed without merging**